docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,257 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Red Hat Single Sign-On
+{ProductShortName} creates a basic
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] instance, using the Keycloak CR of Red Hat Single
+Sign-On Operator. Keycloak is a single sign-on solution for web apps and
+RESTful web services.
+
+The {ProductShortName} Keycloak realm is fully configured by the Hub.
+
+{ProductShortName} does not enable RHSSO external access by default. However, if you
+require RHSSO, you can configure it by using the Tackle CR attribute
```

Comment:
How? rhsso_external_access doesn't even appear listed in "3.2. Installing the Migration Toolkit for Applications Operator and the User Interface".

Reply:
I believe this might be because is a setting we don't advertise, seems to me this is more of a need to know basis depending on use case, my 2c

Reply:
@rromannissen how about we create a ticket to cover the use case mentioned by Franco, and also  include the `rhsso_external_access` operator in the **Migration Toolkit for Applications Operator and the User Interface**.

Reply:
I don't think we are talking about a separated use case there. That's exactly what configuring RHSSO in the context of MTA means. Do we need to keep separated tickets for things that fall in the same domain? @jortel I'd be curious to know your thoughts about this.

Reply:
Sorry I thought i would create a ticket to include the rhsso_external_access operator in the **[Table 3.2. Tackle CR settings](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.0/html-single/user_interface_guide/index#installing_the_migration_toolkit_for_applications_operator_and_the_user_interface)**

Then a second ticket to describe a situation which is *... more of a need to know basis depending on use case*

Apologies if I have this wrong